presto-main/src/main/java/io/prestosql/sql/analyzer/FeaturesConfig.java

```diff
@@ -215,6 +216,18 @@ public FeaturesConfig setDistributedIndexJoinsEnabled(boolean distributedIndexJo
         return this;
     }
 
+    public boolean isLegacyTimestampFormatEnabled()
+    {
+        return enableLegacyTimestampFormat;
+    }
+
+    @Config("legacy-timestamp-format-enabled")
```

Comment:
See comment above. Also, let's prefix the config option with `deprecated.` to make it clear this is on its way out.

Reply:
Fixed